packages/actions/scoreboard/src/modules/scoreboard/sagas.spec.js

```diff
@@ -0,0 +1,75 @@
+import {expectSaga} from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import {all, select, takeEvery} from 'redux-saga/effects'
+import {rest} from 'tocco-app-extensions'
+
+import * as actions from './actions'
+import rootSaga, * as sagas from './sagas'
+
+describe('scoreboard', () => {
+  describe('modules', () => {
+    describe('sagas', () => {
```

Comment:
suggestion(non-blocking): the "describe-path" is not 100% correct

it should reflect the folder path (omit `packages/actions|core|apps|widgets|...` and `src`)
this would result into `scoreboard`, `modules`, `scoreboard`, `sagas`